detectors/src/main/java/io/jshift/odo/detectors/service/JdbcUrlParser.java

```diff
@@ -0,0 +1,40 @@
+package io.jshift.odo.detectors.service;
+
+public class JdbcUrlParser {
+
+    public static String getDatabase(String jdbcUrl) {
+
+        if (jdbcUrl.contains("postgresql")) {
+            return parsePostgreSql(jdbcUrl);
+        } else {
+            if (jdbcUrl.contains("mysql")) {
+                return parseMySql(jdbcUrl);
+            } else {
+                return "";
+            }
+        }
+    }
+
+    private static String parseMySql(String jdbcUrl) {
```

Comment:
These two methods do pretty much the same thing, can we rename it so that it can be used both mysql and postgres

Reply:
They literally do the same thing, imo

Reply:
I know they are doing the same but not all drivers do the same, so I wanted to create a method which just shows the purpose of what they are doing. In this way everyone understands that we are parsing mysql or postgresql jdbc url.